scripts/scenario_00_basic.lua

```diff
@@ -62,19 +63,21 @@ function addWave(enemyList, type, a, d)
     end
 end
 
--- Returns a semi-random heading.
--- cnt: A counter, generally between 1 and the number of enemy groups.
--- enemy_group_count: A number of enemy groups, generally set by the scenario type.
+--- Returns a semi-random heading.
+--
+-- @tparam number cnt A counter, generally between 1 and the number of enemy groups.
+-- @tparam number enemy_group_count A number of enemy groups, generally set by the scenario type.
 function setWaveAngle(cnt, enemy_group_count)
```

Comment:
If we are cleaning this up, "setWaveAngle" doesn't set anything. So "calcWaveAngle" would be a better name.

Reply:
I agree, the code is changed in a different pull request.